explanation/rendering-graphics.md

```diff
@@ -0,0 +1,38 @@
+(exp-rendering-graphics)=
+# Rendering graphical output
+
+Anbox Cloud can adapt to your rendering needs by adjusting its behaviour depending on the availability of a GPU and the presence of the `--enable-graphics` flag when you launch an instance.
+
+    amc launch --enable-graphics my-application
+
+The use of `--enable-graphics` flag when you launch an instance decides whether Anbox Cloud renders a visual output.
+
+When this flag is set, if there is a GPU available, Anbox Cloud renders visual output using the GPU. If there is no GPU available, it uses software rendering to render the visual output.
+
+However, if you do not use the `--enable-graphics` flag when launching an instance, Anbox Cloud does not render a visual output.
+
+Now your environment may or may not include a GPU. If Anbox Cloud detects a GPU device during deployment, it configures the cluster to use the GPU. AMS configures each LXD instance to pass through a GPU device from the host. Currently, all GPUs that are available to a machine are passed to every instance that owns a GPU slot. For NVIDIA GPUs, LXD uses the [NVIDIA container runtime](https://github.com/NVIDIA/nvidia-container-runtime) to make the GPU driver of the host available to the instance.
+
+```{important}
+Anbox Cloud currently does not support GPU provisioning for virtual machines. This feature is planned for a future release.
```

Comment:
This can be read in different ways. If the virtual machine has a GPU attached and we run Anbox Cloud on top of it, we can use it. I think what you mean is that if we run a VM instance within AMS with an Anbox image, right?

Reply:
Yes, the instances that are of type VM and not container - clarified the same.